nautobot_golden_config/templates/nautobot_golden_config/goldenconfigsetting.html

```diff
@@ -190,7 +190,11 @@ <h1>{{ object }}</h1>
                             <tr>
                                 <td>GraphQL Query</td>
                                 <td>
-                                    <pre>{{ object.sot_agg_query|render_json }}</pre>
+                                    {% if object.sot_agg_query %}
+                                        <a href="{{ object.sot_agg_query.get_absolute_url }}">{{ object.sot_agg_query }}</a>
```

Comment:
Is this removing the query value from showing here?

Reply:
Yes. This is replacing it with the name of the Saved Query and a link to the object.

Reply:
sorry, this is in the settings... I was thinking this was in the UI.